.ocamlformat

```diff
@@ -13,3 +13,4 @@ space-around-lists
 space-around-arrays
 cases-exp-indent=2
 break-cases=all
+indicate-nested-or-patterns=unsafe-no
```

Comment:
any idea why it's named `unsafe-no`?

Reply:
I can't say I understood it, but the doc says : 

--indicate-nested-or-patterns={space|unsafe-no}

Control whether or not to indicate nested or-pattern using indentation.
- space starts lines of nested or-patterns with " |" rather than "| ".
- unsafe-no does not indicate nested or-patterns.
Warning: this can produce confusing code where a short body of a match case is visually hidden by surrounding long patterns, leading to misassociation between patterns and body expressions. The default value is space.